bdb/add.c

```diff
@@ -91,7 +91,8 @@ static int bdb_prim_addkey_int(bdb_state_type *bdb_state, tran_type *tran,
         abort();
     }
 
-    rrn = 2;
+    /* JJM 2018-05-02: This value is not actually used by this function. */
+    /* rrn = 2; */
```

Comment:
I thought about doing that; however, since somebody mentioned that the value is always 2, I thought perhaps leaving it as a comment would be good for future code archaeology.